datafusion/sqllogictest/test_files/repartition_scan.slt

```diff
@@ -147,7 +147,7 @@ WITH HEADER ROW
 LOCATION 'test_files/scratch/repartition_scan/csv_table/';
 
 query I
-select * from csv_table;
+select * from csv_table ORDER BY column1;
```

Comment:
This makes the test result stable, otherwise every time I rerun it locally, it gets different results...